fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/BasicCacheEntry.java

```diff
@@ -78,7 +78,7 @@
                                     calculatedChecksum,
                                     algorithm);
 
-            return asList(result);
+            return Collections.singletonList(result);
```

Comment:
For consistency with what was replaced and usage elsewhere, this should probably be a static import